estuary/api/v1.py

```diff
@@ -276,7 +276,18 @@ def get_siblings(resource, uid):
         serialized_node['resource_type'] = inflated_node.__label__
         serialized_results.append(serialized_node)
 
-    return jsonify(serialized_results)
+    item_type = item.__label__
+    item_id = item.id_
```

Comment:
Creation of new variables, `item_type` and `item_id` seem unnecessary to me. Instead, you can directly pass them